docs/static-pvc.md

```diff
@@ -0,0 +1,240 @@
+# Static PVC with ceph-csi
+
+- [Static PVC with ceph-csi](#static-pvc-with-ceph-csi)
+  - [RBD static volume](#rbd-static-volume)
+    - [Create static RBD Filesystem PV](#create-static-rbd-filesystem-pv)
+    - [Create static RBD Filesystem PVC](#create-static-rbd-filesystem-pvc)
+    - [Create app pod to use the Filesystem PVC](#create-app-pod-to-use-the-filesystem-pvc)
+    - [Delete Resources created for static Filesystem PV](#delete-resources-created-for-static-filesystem-pv)
+    - [Create static RBD Block PV](#create-static-rbd-block-pv)
+    - [Create static RBD Block PVC](#create-static-rbd-block-pvc)
+    - [Create app pod to use the Block PVC](#create-app-pod-to-use-the-block-pvc)
+    - [Delete Resources created for static Block PV](#delete-resources-created-for-static-block-pv)
+
+This document outlines how to use static PVC and static PV for rbd CSI driver
+
+**warning** static PVC can be created, deleted, mounted and unmounted but
+doesn't support other operations like (snapshot, resize, etc)
+
+## RBD static volume
+
+rbd images created manually can be mounted and unmounted to an app, below step
+shows how to create a static PV, static PVC and bind PVC to an application
+
+### Create static RBD Filesystem PV
+
+to create the rbd PV you need to know the `rbd image name`,`clusterID` and
+`pool` name in which the rbd image is created
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: fs-static-pv
+spec:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 4Gi
+  csi:
+    driver: rbd.csi.ceph.com
+    fsType: ext4
+    nodeStageSecretRef:
+      # node stage secret name
+      name: rook-csi-rbd-node
+      # node stage secret namespace where above secret is created
+      namespace: rook-ceph
+    volumeAttributes:
```

Comment:
I think we need to leave a note about `volumeAttributes` and that all parameters in the storage class can be added here.